.github/workflows/ci-cd.yml

```
@@ -21,6 +21,10 @@ jobs:
   lint:
     name: 코드 스타일 검사
     runs-on: ubuntu-latest
+    # PR 또는 직접 push 시에만 실행 (PR 머지로 인한 push는 제외)
+    if: |
+      github.event_name == 'pull_request' || 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)